src/Uno.UI/UI/Xaml/Controls/ContentControl/ContentControl.wasm.cs

```diff
@@ -15,6 +15,13 @@ namespace Windows.UI.Xaml.Controls
 {
 	public partial class ContentControl
 	{
+		public ContentControl(string htmlTag) : base(htmlTag)
+		{
+			DefaultStyleKey = typeof(ContentControl);
+
+			InitializePartial();
+		}
```

Comment:
Seems odd that it's missing. @davidjohnoliver  ?

Reply:
It's not missing. It's just I added another constructor for the html tag :-)